synapse/module_api/__init__.py

```diff
@@ -1452,6 +1452,28 @@ async def get_monthly_active_users_by_service(
             start_timestamp, end_timestamp
         )
 
+    async def lookup_room_alias(self, room_alias: str) -> Tuple[str, List[str]]:
+        """
+        Get the room ID associated with a room alias.
+
+        Added in Synapse v1.65.0.
+
+        Args:
+            room_alias: The alias to look up.
+        Returns:
+            A tuple of:
+                The room ID (str).
+                Hosts likely to be participating in the room ([str]).
+        Raises:
+            SynapseError if room alias is invalid or could not be found.
```

Comment:
```suggestion
        Args:
            room_alias: The alias to look up.

        Returns:
            A tuple of:
                The room ID (str).
                Hosts likely to be participating in the room ([str]).

        Raises:
            SynapseError if room alias is invalid or could not be found.
```

Spacing out for readability.

Reply:
Done:

- 1e217ec
- 0d5ff6c